### PR TITLE
Add scripts to make local running on linux easier

### PR DESCRIPTION
**Pull request by jebbinSCL**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -62,3 +62,9 @@ The secrets.config file should contain:
     <add key="AWSSecretKey" value="" />
 </appSettings>
 ```
+
+## Running on linux.
+To run easily on linux for local development you need to
+1. Edit the app.config file if required.
```
Comment (pallenSL) [inline README.md]: I don't think your numbering is working.  Try "View".

File: localBuildRun.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+#!/bin/bash -euv
+echo -e '\033]2;'Comms'\007'
```
Comment (pallenSL) [inline localBuildRun.sh]: I don't think the control characters are wise in a script.

Comment (jebbinSCL) [inline localBuildRun.sh]: These are only to set the window title to 'Comms'

**Comment by pallenSL** [comment]
:shipit:

**Comment by dmorrisSL** [comment]
:shipit:

**Pull request merged**